cmd/tier-handlers.go

```diff
@@ -202,11 +211,14 @@ func (api adminAPIHandlers) RemoveTierHandler(w http.ResponseWriter, r *http.Req
 
 	defer logger.AuditLog(ctx, w, r, mustGetClaimsFromToken(r))
 
-	objAPI, _ := validateAdminReq(ctx, w, r, iampolicy.SetTierAction)
-	if objAPI == nil || globalNotificationSys == nil || globalTierConfigMgr == nil {
+	if globalTierConfigMgr == nil {
```

Comment:
`globalTierConfigMgr` is [initialized](https://github.com/minio/minio/blob/master/cmd/server-main.go#L366) to a non-nil value before the object layer is [initialized](https://github.com/minio/minio/blob/master/cmd/server-main.go#L416). So, we don't need to check `if globalTierConfigMgr == nil` in any of the tier handlers. I realized this only on reviewing this PR.